x-pack/plugins/security_solution/public/common/containers/alerts/use_alert_prevalence_from_process_tree.ts

```diff
@@ -36,20 +21,91 @@ interface ProcessTreeAlertPrevalenceResponse {
   alertIds: string[];
 }
 
-export function useAlertPrevalenceFromProcessTreeActual(
-  processEntityId: string
+interface EntityResponse {
+  id: string;
+  name: string;
+  schema: object;
+}
+
+interface TreeResponse {
+  statsNodes: Array<{
+    data: object;
+    id: string;
+    parent: string;
+    stats: {
+      total: number;
+      byCategory: {
+        alerts?: number;
+      };
+    };
+  }>;
+  alertIds: string[];
+}
+
+function useAlertDocumentAnalyzerSchema(processEntityId: string) {
+  const http = useHttp();
+  const query = useQuery<EntityResponse[]>(['getAlertPrevalenceSchema', processEntityId], () => {
+    return http.get<EntityResponse[]>(`/api/endpoint/resolver/entity`, {
+      query: {
+        _id: processEntityId,
+        indices: ['.alerts-security.alerts-default', 'logs-*'],
```

Comment:
I would alias this to use the spaceId here `.alerts_security.alerts-{spaceId}`. Maybe re-purpose this? https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/public/risk_score/containers/common/index.ts

Reply:
I meant to use the new hook I made for this and just completely forgot ha, thanks for finding